### ISSUE TITLE
failed to find document avr/io.h

### ISSUE THREAD

**Issue by Somnus0737** [madsci1016/Arduino-EasyTransfer#18]
Oh,hello.I followed the guidance to put SoftEasyTranfer.h in the libraries,but the Arduino IDE couldn't find the avr/io.h when I compiled my project.What's the problem?Could you give me some advices?

**Comment by per1234** [comment]
Which board do you have selected in the **Tools > Board** menu?

**Comment by Somnus0737** [comment]
nodemcu

**Comment by Somnus0737** [comment]
Actually,I want to connect nodemcu to arduino uno by EasyTransfer

**Comment by beicnet** [comment]
@per1234 Can you help us with this?

I wish to run only I2C communication on **WeMos D1 mini** as master and **Arduino Uno** as slave, but like above described, I got compile error for the ESP8266 platform too, "fatal error: avr/io.h: No such file or directory"

**Comment by beicnet** [comment]
@per1234 
Yes, commenting it out did the trick.
But, the example for the I2C not working for me.
They are compiling fine, but the slave device (Uno) is not apearing on the I2C bus.
I2C scanner says, "No I2C devices found", even tried with 2x Unos.

Any help would be appreciated!

**Comment by per1234** [comment]
Talking about an potentially unnecessary incompatibility with non-AVR architectures is reasonable for an issue tracker but now this is going more in the direction of a support request for your project. Further discussion on that topic is probably more appropriate for the Arduino Forum:
http://forum.arduino.cc/

**Comment by beicnet** [comment]
@per1234 Yes, I know that, but the included I2C examples are not working even with 2x Arduino Unos.

**Comment by ShadGates** [comment]
FYI: EasyTransferI2C.h has the same problem, a lot of un-needed includes at the top of the file, including the avr/io.h mentioned in this issue.
